news_items/005_fractions_and_input_changes/index.html

```diff
@@ -0,0 +1,81 @@
+
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS '21)
```

Comment:
```suggestion
  <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS BS '19, CS MS '21)
```

Reply:
Looks good, I think @nicknytko can commit the suggestion?